client/app/components/conference_room/provider/EventProvider.jsx

```diff
@@ -16,22 +16,34 @@ export default class EventProvider extends React.Component {
   constructor(...args) {
     super(...args);
     this.state = { nextEvents: [] };
-    bindAll(this, ['_fetchForToday', 'handleUpdate', '_confirmEvent']);
+    bindAll(this, ['_fetchForToday', 'handleUpdate', '_confirmEvent', 'updateEvents']);
   }
 
   componentDidMount() {
     this._fetchForToday();
     this._channelSubscription = createSubscription(EVENT_CHANNEL, this._fetchForToday);
+    this.setUpdateTimeout();
   }
 
   componentWillUnmount() {
     removeSubscription(this._channelSubscription);
+    window.clearTimeout(this.updateTimeout);
   }
 
   handleUpdate() {
     this._fetchForToday();
   }
 
+  updateEvents() {
+    this._fetchForToday();
+    this.setUpdateTimeout();
+  }
+
+  setUpdateTimeout() {
+    const now = moment();
+    this.updateTimeout = setTimeout(this.updateEvents, now.clone().add(1, 'day').startOf('day') - now);
```

Comment:
I would extract `now.clone().add(1, 'day').startOf('day') - now` to variable and name it beginningOfTheNextDay. Maybe moment already has a function for that?